src/Constants.ts

```diff
@@ -517,6 +523,7 @@ export const SpanTags = {
     TRIGGER_OPERATION_NAMES: 'trigger.operationNames',
     TOPOLOGY_VERTEX: 'topology.vertex',
     TRACE_LINKS: 'trace.links',
+    SPAN_RESOURCES: 'span.resources',
```

Comment:
I think, the constant name should be `RESOURCE_NAMES` and tag name should be `resource.names` accordingly